.storybook/main.js

```diff
@@ -1,7 +1,7 @@
 const { resolve } = require('path');
 
 module.exports = {
-  stories: ['../src/**/*.stories.(js|mdx)'],
+  stories: ['../src/welcome.stories.mdx', '../src/**/*.stories.(js|mdx)'],
```

Comment:
Can you add a comment explaining that this is intended to control the order? I'm aware of three methods for controlling the nav order so it would be helpful to know that is the intention here.